webui/react/src/components/ThemeToggle.test.tsx

```diff
@@ -1,7 +1,7 @@
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import React from 'react';
-import { Router } from 'react-router-dom';
+import { unstable_HistoryRouter as HistoryRouter } from 'react-router-dom';
```

Comment:
is there a documentation URL about this? would be good to make a comment or know if this is being deprecated

Reply:
Yes, https://reactrouter.com/en/v6.3.0/api#unstable_historyrouter
We need to use this to use `history` lib